### PR TITLE
Hide similar events from eventpage

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
See conversation here: https://helsinkicity.slack.com/archives/C013682SYSD/p1612348876019900
Similar events with lots of canceled events makes marketing campaigns look not so good. Thus similar events will be hidden for the time being. It's done behind feature flag, also introduced crude "karvalakki" version of the feature flag functionality. 
Also added tests that check that feature flag works on and off.
## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/domain/event/__tests__/EventPageContainer.test.tsx [hunk 1]
```diff
@@ -100,3 +106,32 @@ it("should show error info when event doesn't exist", async () => {
...
+    await waitFor(() => {
+      expect(screen.queryByTestId('loading-spinner')).not.toBeInTheDocument();
+    });
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.similarEvents.title,
+      })
+    ).toBeInTheDocument();
+  });
+  it('doesnt shor similar events when flag is off', async () => {
```
Comment (Sartonon) [inline src/domain/event/__tests__/EventPageContainer.test.tsx:124]: typo: shor

File: src/domain/event/EventPageContainer.tsx [hunk 2]
```diff
@@ -53,7 +53,9 @@ const EventPageContainer: React.FC = () => {
               {/* Show event content only if event is open */}
               {!eventClosed && <EventContent event={event} />}
               {/* Hide similar event on SSR to make initial load faster */}
-              {isClient && <SimilarEvents event={event} />}
+              {isClient && getFeatureFlags().SHOW_SIMILAR_EVENTS && (
```
Comment (Sartonon) [inline src/domain/event/EventPageContainer.tsx:56]: Minor readability improvement would be to call this function in top of this function and destructure SHOW_SIMILAR_EVENTS. But that is debatable :D

**Review by Sartonon — APPROVED** [review]
Looks good. Couple comments but approved.

File: src/domain/event/__tests__/EventPageContainer.test.tsx (hunk 1 above)
Comment (nenonja) [reply src/domain/event/__tests__/EventPageContainer.test.tsx]: fixed

File: src/domain/event/EventPageContainer.tsx (hunk 2 above)
Comment (nenonja) [reply src/domain/event/EventPageContainer.tsx:56]: fixed

**Pull request merged**